docs/HACKING.md

```diff
@@ -69,10 +69,10 @@ another terminal on your host (choose the right one depending on the
 distribution of the container or virtual machine):
 
 ```sh
-mkosi -t none && mkosi ssh dnf upgrade --disablerepo="*" "/work/build/*.rpm" # CentOS/Fedora
-mkosi -t none && mkosi ssh apt install --reinstall "/work/build/*.deb" # Debian/Ubuntu
-mkosi -t none && mkosi ssh pacman -U "/work/build/*.pkg.tar" # Arch Linux
-mkosi -t none && mkosi ssh zypper install --allow-unsigned-rpm "/work/build/*.rpm" # OpenSUSE
+mkosi -t none && mkosi ssh dnf upgrade --disablerepo="*" --assumeyes "/work/build/*.rpm" # CentOS/Fedora
+mkosi -t none && mkosi ssh apt-get install "/work/build/*.deb" # Debian/Ubuntu
```

Comment:
are you sure --reinstall is not needed?

Reply:
Yes I tested and it's not required if the version+release is higher than the installed one

Reply:
Are you sure that's always going to be the case? What about trying to rollback to a previous build? I mean it's a doc so it's not a big issue either way

Reply:
Well this doc is specifically about iterating by building a new version with mkosi and installing that one. Those versions are guaranteed to be newer.